jf_agent/data_manifests/git/adapters/github.py

```diff
@@ -21,15 +21,28 @@ class GithubManifestGenerator(ManifestAdapter):
     '''
 
     def __init__(
-        self, token: str, company: str, org: str, instance: str, verify=True, **kwargs
+        self,
+        token: str,
+        base_url: str,
+        company: str,
+        org: str,
+        instance: str,
+        verify=True,
+        **kwargs,
     ) -> None:
         # Super class fields
         self.company = company
         self.org = org
         self.instance = instance
         # Session fields
         self.token = token
-        self.base_url = 'https://api.github.com/graphql'
+
+        # potentially cleanup base url
+        base_url[:-1] if base_url and base_url.endswith('/') else base_url
+        if 'api/v3' in base_url:
+            self.base_url = base_url.replace('api/v3', 'graphql')
+        else:
+            self.base_url = f'{base_url}/graphql' or 'https://api.github.com/graphql'
```

Comment:
B: `f'{base_url}/graphql'` will always be true even if `base_url == None`. What is this trying to accomplish?